src/components/DataTable/_data-table.scss

```diff
@@ -1 +1,62 @@
 @import '~carbon-components/scss/components/data-table/data-table';
+
+.bx--table-header-label {
+  &-start {
+    span.bx--table-header-label {
```

Comment:
if not necessary for specificity we should use the class here without the element selector.

Reply:
Code is updated based on your review comments (removed span), Please review.